@here/harp-text-canvas/lib/rendering/FontCatalog.ts

```diff
@@ -502,6 +503,34 @@ export class FontCatalog {
         })!;
     }
 
+    /**
+     * Update the info with the memory footprint caused by objects owned by the `FontCatalog`.
+     *
+     * @param info The info object to increment with the values from this `FontCatalog`.
+     */
+    updateMemoryUsage(info: MemoryUsage) {
+        let numBytes = 0;
+
+        for (const block of this.unicodeBlocks) {
```

Comment:
This is not really an accurate estimation, as ``unicodeBlocks`` only contains info on which blocks are covered, but a single block could be covered by multiple fonts .

I think the ``FontCatalog`` memory estimation should be something like:
* Sum of all the info passed on construction (like ``unicodeBlocks`` and ``fonts``).
* ``GlyphTextureCache`` size estimation (already accurate).
* Sum of all the arays contained in all ``m_loadedJson``.
* Sum of all the textures in ``m_loadedPages`` (already accurate).
* Sum of all ``m_loadedGlyphs``.

Reply:
The main memory hogs are already accounted for. For improved precision, we can modify later.